_docs/additional-features.md

```diff
@@ -447,3 +447,70 @@ The endpoint responds with a list of uploaded proof of payment files, e.g.:
 {% endcapture %}
 
 {% include language-tabbar.html prefix="proof-of-payments" raw=data-raw %}
+
+# Balance enquiry
+
+To obtain your current account balance you can use following two endpoints:
+
+`GET /v1/accounts` to obtain your balance for all currencies or
+
+`GET /v1/accounts/[CURRENCY_NAME]` to obtain for a single currency.
+
+The response in both cases look like the following (with only one currency returned in the latter instance)
+
+```javascript
+{
+  "meta": {
+    "negative_balance": false // returns if you are allowed to go beyond your account balance
+  },
+  "object": [
+    {
+      "amount": 1000,
+      "currency": "USD"
+    },
+    {
+      "amount": 1000,
+      "currency": "XOF"
+    }
+    // [...]
+  ]
+}
+```
+
+<div class="alert alert-warning" markdown="1">
+**Warning!** Refrain from using this endpoint more than once in every five minutes.
+</div>
+
+# Get current exchange rates
+
+To obtain the current exchange rates in the system you can call the following endpoint:
+
+`GET /v1/info/currencies/in`
+
+The response will contain all possible currency pairs. For example the current rate to convert funds from `NGN` to `XOF` will appear the following way:
```

Comment:
@arnaugiro I think we want to put in details here about the inbound and outbound pricing we have been discussing.

Reply:
The results here are definitely made simpler, and lacks a lot of details we have, like margin and fees. Once we have a decision on how to handle and show this to customers it would be useful to add it here